data/yam/agama/auto/sles_rmt_ssh_pub_key_ppc64le.jsonnet

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
now should be possible to use the template also for remote architectures, so we can drop this file.

Reply:
Removed and use template.